### PR TITLE
SIDM-9020-Added Accessibility tests to idam-user-dashboard

### PR DESCRIPTION
**Pull request by chandkanuri**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/a11y/amendAccessibilityReport.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path');
+
+
+function amendAccessibilityReport() {
+    amendHeading( /manage-user-a11y-audit\.html$/, 'AXE Accessibility Results - Manage User Page');
+    amendHeading(/search-user-a11y-audit\.html$/, 'AXE Accessibility Results - Search User Page');
+    amendHeading(/edit-user-a11y-audit\.html$/, 'AXE Accessibility Results - Edit User Page');
+    amendHeading(/generate-user-report-a11y-audit\.html$/, 'AXE Accessibility Results - Generate User Report Page');
+    amendHeading(/add-new-user-a11y-audit\.html$/, 'AXE Accessibility Results - Add New User Page');
```
Comment (jburke-idam) [inline src/test/a11y/amendAccessibilityReport.js:10]: bit hardcoded - might be better to just put the file name in the header (maybe without the "html")

File: src/test/v2/accessibility/accessibility_test.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Before(async ({ setupDAO, login }) => {
+  await setupDAO.setupAdmin();
+  login('admin');
+});
+
+Scenario('I am on manage user page',  async ({ I }) => {
+  const testUser = await I.haveUser();
+  await I.navigateToManageUser(testUser.email);
+
+  I.runA11yCheck({ reportFileName: 'manage-user-a11y-audit.html' });
```
[the remark this answers is not in the document]
Comment (chandkanuri) [reply src/test/v2/accessibility/accessibility_test.ts]: Done

File: src/test/a11y/amendAccessibilityReport.js (hunk 1 above)
Comment (chandkanuri) [reply src/test/a11y/amendAccessibilityReport.js]: Done

File: src/test/v2/common/steps_file.ts [hunk 3]
```diff
@@ -113,5 +114,11 @@ export = function() {
       }
       return rsp;
     },
+    checkA11y(I: any, fileName: string) {
+      I.runA11yCheck({ reportFileName: fileName });
```
Comment (jburke-idam) [inline src/test/v2/common/steps_file.ts:118]: You shouldn't be passing "I" in here - all references to I should work by just using "this."

**Review by jburke-idam — APPROVED** [review]

**Pull request merged**